Sources/Sham/MockService.swift

```diff
@@ -29,7 +29,7 @@ public class MockService {
     }()
     
     /// A dictionary of stubbed responses keyed by stubbed requests.
-    var stubbedDataCollection = StubbedDataCollection()
+    public var stubbedDataCollection = StubbedDataCollection()
```

Comment:
I was originally gonna comment when this lost it's `private` marker, but didn't since it was `internal`. But I'm not sure we'd like this to be public. I feel like we'd rather have total control over how this is changed through the methods to add and clear stubbed data.

Reply:
This crossed my mind. On the `StubbedDataCollection` object, the stubbed data is private and controlled by methods. I felt that that covered this concern. Any managing functions on `MockService` are a formality to be backwards compatible with these changes, and the information is passed on to the `StubbedDataCollection` to handle anyway.

Reply:
I'm still not sure we'd want people changing this entire object on the mock service. If we wanted to open it up a bit, I think making it `private(set)` and removing the wrapper methods to set/clear data would be better, but not sure that makes sense in this PR given that would be a breaking change to `MockService`. If we wanted to make that change, that should probably be it's own ticket and a new release.

Reply:
What do you feel makes sense for this PR? Creating a ticket and an associated TODO?

Reply:
Probably changing it back to `private` so access to it remains through the `MockService` and creating a ticket for changing how one stubs data. No real need to add a TODO since it's not really like we're missing functionality that needs completed before things work properly.